benchmark/utils/setup.ts

```diff
@@ -14,7 +14,7 @@ export interface Contracts {
 }
 
 const generateTarget = async (owners: Wallet[], threshold: number, guardAddress: string, logGasUsage?: boolean) => {
-    const fallbackHandler = await getDefaultCallbackHandler()
+    const fallbackHandler = await getTokenCallbackHandler()
```

Comment:
just spotted that is uses the token callback handler and not the compatibility one. Should this be fixed?